platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/AttributionDialogManager.java

```diff
@@ -0,0 +1,161 @@
+package com.mapbox.mapboxsdk.maps;
+
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.net.Uri;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AlertDialog;
+import android.text.Html;
+import android.text.SpannableStringBuilder;
+import android.text.TextUtils;
+import android.text.style.URLSpan;
+import android.view.View;
+import android.widget.ArrayAdapter;
+
+import com.mapbox.mapboxsdk.R;
+import com.mapbox.mapboxsdk.camera.CameraPosition;
+import com.mapbox.mapboxsdk.style.sources.Source;
+import com.mapbox.services.android.telemetry.MapboxTelemetry;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Locale;
+
+class AttributionDialogManager implements View.OnClickListener, DialogInterface.OnClickListener {
```

Comment:
done in https://github.com/mapbox/mapbox-gl-native/pull/8630/commits/527ffbe733979f393dafd23a062d895e524ea294#diff-8a5300c6fbc51b88f2e971461adcb513R33